frontend/src/app/supportAdmin/pageTitles.ts

```diff
@@ -3,6 +3,7 @@ import { PATIENT_TERM_PLURAL } from "../../config/constants";
 // js file so imports to cypress work
 export const addNewDevicePageTitle = "Add new device";
 export const editDevicePageTitle = "Edit existing device";
+export const manageUserPageTitle = "Manage users";
```

Comment:
ooo I totally forgot this file existed... thank you for adding it here so I know what to update in my PR 😆

Reply:
+1